parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -296,10 +296,7 @@ try
 
     # NOTE: this function MUST be called before Write-KubeClusterConfig since it has the potential
     # to mutate both kubelet config args and kubelet node labels.
-    Disable-KubeletServingCertificateRotationForTags
-
-    # TODO(cameissner): uncomment once CSE package has been updated
-    # Configure-KubeletServingCertificateRotation
+    Configure-KubeletServingCertificateRotation

```

Comment:
The comment above - does it apply to both functions (i mean the one you removed and tthis new uncommented one)?

Reply:
yep, that's why the comment's staying in